### PR TITLE
Fix `M862.x` LCD messages

### PR DESCRIPTION
**Pull request by 3d-gussner**
As the new menue adds a Yes/No option the messages need to be limited to c=20 r=3
Updated/shortened all translations

- [X] Tested on MK404 in English and French (usually longest messages)

### CODE REVIEW

File: Firmware/messages.cpp [hunk 1]
```diff
@@ -133,8 +133,14 @@ const char MSG_NONE[] PROGMEM_I1 = ISTR("None"); ////MSG_NONE c=8
...
-const char MSG_GCODE_DIFF_PRINTER_CONTINUE[] PROGMEM_I1 = ISTR("G-code sliced for a different printer type. Continue?"); ////MSG_GCODE_DIFF_PRINTER_CONTINUE c=20 r=5
+const char MSG_GCODE_DIFF_PRINTER_CONTINUE[] PROGMEM_I1 = ISTR("G-code sliced for a different printer type. Continue?"); ////MSG_GCODE_DIFF_PRINTER_CONTINUE c=20 r=3
 const char MSG_GCODE_DIFF_PRINTER_CANCELLED[] PROGMEM_I1 =ISTR("G-code sliced for a different printer type. Please re-slice the model again. Print cancelled."); ////MSG_GCODE_DIFF_PRINTER_CANCELLED c=20 r=8
+const char MSG_GCODE_NEWER_FIRMWARE_CONTINUE[] PROGMEM_I1 = ISTR("G-code sliced for a different printer type. Continue?"); ////MSG_GCODE_NEWER_FIRMWARE_CONTINUE c=20 r=3
+const char MSG_GCODE_NEWER_FIRMWARE_CANCELLED[] PROGMEM_I1 = ISTR("G-code sliced for a different printer type. Continue?"); ////MSG_GCODE_NEWER_FIRMWARE_CANCELLED c=20 r=8
+const char MSG_GCODE_DIFF_CONTINUE[] PROGMEM_I1 = ISTR("G-code sliced for a different printer type. Continue?"); ////MSG_GCODE_DIFF_CONTINUE c=20 r=3
+const char MSG_GCODE_DIFF_CANCELLED[] PROGMEM_I1 = ISTR("G-code sliced for a different printer type. Continue?"); ////MSG_GCODE_DIFF_CANCELLED c=20 r=8
```
Comment (gudnimg) [inline Firmware/messages.cpp]: Should be `"G-code sliced for a different printer type. Please re-slice the model again. Print cancelled."`

File: Firmware/messages.cpp [hunk 2]
```diff
@@ -133,8 +133,14 @@ const char MSG_NONE[] PROGMEM_I1 = ISTR("None"); ////MSG_NONE c=8
 const char MSG_WARN[] PROGMEM_I1 = ISTR("Warn"); ////MSG_WARN c=8
 const char MSG_STRICT[] PROGMEM_I1 = ISTR("Strict"); ////MSG_STRICT c=8
 const char MSG_MODEL[] PROGMEM_I1 = ISTR("Model"); ////MSG_MODEL c=8
-const char MSG_GCODE_DIFF_PRINTER_CONTINUE[] PROGMEM_I1 = ISTR("G-code sliced for a different printer type. Continue?"); ////MSG_GCODE_DIFF_PRINTER_CONTINUE c=20 r=5
+const char MSG_GCODE_DIFF_PRINTER_CONTINUE[] PROGMEM_I1 = ISTR("G-code sliced for a different printer type. Continue?"); ////MSG_GCODE_DIFF_PRINTER_CONTINUE c=20 r=3
 const char MSG_GCODE_DIFF_PRINTER_CANCELLED[] PROGMEM_I1 =ISTR("G-code sliced for a different printer type. Please re-slice the model again. Print cancelled."); ////MSG_GCODE_DIFF_PRINTER_CANCELLED c=20 r=8
+const char MSG_GCODE_NEWER_FIRMWARE_CONTINUE[] PROGMEM_I1 = ISTR("G-code sliced for a different printer type. Continue?"); ////MSG_GCODE_NEWER_FIRMWARE_CONTINUE c=20 r=3
+const char MSG_GCODE_NEWER_FIRMWARE_CANCELLED[] PROGMEM_I1 = ISTR("G-code sliced for a different printer type. Continue?"); ////MSG_GCODE_NEWER_FIRMWARE_CANCELLED c=20 r=8
```
Comment (gudnimg) [inline Firmware/messages.cpp]: Should be `"G-code sliced for a newer firmware. Please update the firmware. Print cancelled."`

**Review by gudnimg — CHANGES_REQUESTED** [review]
PR looks good to me except two of the messages.

File: Firmware/messages.cpp (hunk 2 above)
Comment (gudnimg) [reply Firmware/messages.cpp]: `MSG_GCODE_NEWER_FIRMWARE_CONTINUE` should also be `"G-code sliced for a newer firmware. Continue?"`

File: Firmware/messages.cpp (hunk 1 above)
Comment (3d-gussner) [reply Firmware/messages.cpp]: Thanks copy/paste typo

File: Firmware/messages.cpp (hunk 2 above)
Comment (3d-gussner) [reply Firmware/messages.cpp]: Thanks copy/paste typo

**Review by DRracer — APPROVED** [review]

**Pull request merged**